multimedia/minisatip/files/minisatip.init

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+
```

Comment:
You may want to define `START=N` so that `/etc/init.d/minisatip enable` knows how to setup symlinks for autostart at boot